docs/gluster-block.8

```diff
@@ -134,9 +137,12 @@ To create a thinly-provisioned block device of size 1GiB
 To create a block device with existing file in blockVol
 .B # gluster-block create blockVol/sampleBlock storage 3c242959-9238-4ce0-a1ed-19086aa23c7b ${HOST}
 
-To create a block device with ring buffer size 16MB
+To create a block device with ring buffer size 1GiB
```

Comment:
@lxbsz should this be 16MB only ?

Reply:
Yeah, sorry, make one mistake as the size.
Will fix it.